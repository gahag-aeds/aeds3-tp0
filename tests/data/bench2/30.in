30 15
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 
Add 5 14
Max 12 21
Add 1 10
Min 16 25
Sub 4 13
Min 7 16
Add 13 22
Min 12 21
Sub 6 15
Min 4 13
Add 11 20
Sum 10 19
Add 5 14
Sum 13 22
Sub 10 19
