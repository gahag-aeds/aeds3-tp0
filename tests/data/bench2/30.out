67
10
10
11
17
422
426
